README.md

```diff
@@ -1 +1,27 @@
-# gapminder_dashboard
\ No newline at end of file
+# Gapminder dashboard
+
+## Decription of the app
+
+Gapminder data set is a great source to know about the world. So The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.
+
+The landing page of the dashboard is designed to have five filters at the left, for selecting the target of the study, second factor of study, continent , country and the year range (1960 to 2018) .The line chart at the top is a trend-plot showing the change of the target of the study during the time over the years for the selected country, average of the continent and average of the whole world. So user would be able to compare these trend with each other. In the top right corner, there is a bubble chart which visualize the target study versus the second factor of study for all the countries.
```

Comment:
The landing page of the dashboard is designed to have a panel of five filters for which users can interact with. These filters include selecting a primary and secondary target to study, a continent, a country and the year range of interest (1960 to 2018). The line chart is a trend-plot showing the change of the primary target of the study over the time interval for a selected country. It also includes the average of the continent and average of the whole world which gives users the ability to compare trends.
Additionally we include a bubble chart which visualizes the target study versus the second factor of study for all the countries.